gensim/models/poincare.py

```diff
@@ -91,6 +91,8 @@ def __init__(self, train_data, size=50, alpha=0.1, negative=10, workers=1, epsil
             Number of threads to use for training the model.
         epsilon : float, optional
             Constant used for clipping embeddings below a norm of one.
+        regularization_coeff : float, optional
```

Comment:
Can the L2 regularization be disabled? Setting `regularization_coeff = 0.0` should do it, correct? Might be worth mentioning here.

Reply:
That's correct. Good point, adding.